src/Collections/Plans.php

```diff
@@ -18,7 +18,7 @@ class Plans extends SiteOwnedCollection
     /**
      * @var string
      */
-    protected $url = 'sites/{site_id}/plans';
+    protected $url = 'accounts/site-account-forwarding/{site_id}/plans';
```

Comment:
This is not intended to be permanent and isn't a great pattern to codify in Terminus. I wonder if we can just make the old `sites/id/plans` API work the same way for now until we sort out how accounts should be handled in the UI and in Terminus